src/imgix-core-js.js

```diff
@@ -124,6 +124,74 @@
       }
     };
 
+    ImgixClient.prototype.buildSrcSet = function (path, params) {
+      var width = params ? params['w'] : undefined;
+      var height = params ? params['h'] : undefined;
+      var aspectRatio = params ? params['ar'] : undefined;
+
+      // determines if an aspect ratio value is in the correct format 'w:h'
+      var isValidFormat = function() {
```

Comment:
I'm not sure if it's ok stylistically to have anonymous functions littered throughout the code. The main reasons I chose to do it this way was for readability and to avoid exposing certain private functions on the `ImgixClient.prototype`

Reply:
I'd say you're not gaining anything by using a function here. I'd just do this:

```js
var isValidArFormat = /^\d+(\.\d+)?:\d+(\.\d+)?$/.test(aspectRatio);
```

No unnecessary function scope, no exposed private function, and no litter.